jobs/JRRFS_UNGRIB

```diff
@@ -4,12 +4,26 @@ set -x
 date
 #
 # source the config cascade
-source ${EXPDIR}/exp.setup
-[[ ! -z "${ENS_INDEX}" ]] && source ${EXPDIR}/config/config.ens
-source ${EXPDIR}/config/config.${MACHINE}
-source ${EXPDIR}/config/config.base
-[[ -s ${EXPDIR}/config/config.ungrib ]] && source ${EXPDIR}/config/config.ungrib
+#source ${EXPDIR}/exp.setup
+#[[ ! -z "${ENS_INDEX}" ]] && source ${EXPDIR}/config/config.ens
+#source ${EXPDIR}/config/config.${MACHINE}
+#source ${EXPDIR}/config/config.base
+#[[ -s ${EXPDIR}/config/config.ungrib ]] && source ${EXPDIR}/config/config.ungrib
```

Comment:
The most of the environmental setups should come from xml/ecflow instead of configuration files. Those configuration files indue too many environmental variables that are nor needed for the task.

Reply:
I think we need a discussion on this. 
We have a simple system at the moment. As we move forward, there will be many tuning parameters. rrfs.v1 is a good reference on the scale of the tuning parameters. Some tuning parameters are shared across different tasks. We better double check whether we want to set all those options in each individual tasks.